config/locales/en.yml

```diff
@@ -12,7 +12,7 @@ en:
   helpers:
     submit:
       address:
-        create: Use this address
+        create: Continue
```

Comment:
The default button (under `simple_form.helpers.submit.create`) is already `Continue`, so you can remove this key altogether.